### PR TITLE
Accommodate Enclave Hardware Non-Simulation Mode

### PR DESCRIPTION
**Pull request by jasondu7297**

### CODE REVIEW

File: src/host/host.cpp [hunk 1]
```diff
@@ -45,17 +45,21 @@ class RPCHandler : virtual public RPCIf {
...
             spdlog::debug("Running in simulation mode");
             #endif
 
-            oe_result_t result =
-                oe_create_ortoa_enclave(oe_enclave_path, OE_ENCLAVE_TYPE_SGX,
-                                        OE_ENCLAVE_FLAG_SIMULATE, NULL, 0, &enclave);
-            if (result != OE_OK) {
-                throw OECreationFailed(oe_enclave_path);
-            }
+            simulate_flag = OE_ENCLAVE_FLAG_SIMULATE;
```
Comment (adriandavila) [inline src/host/host.cpp:55]: Look at line 46 here https://github.com/openenclave/openenclave/blob/e9a0423e3a0b242bccbe0b5b576e88b640f88f85/samples/pluggable_allocator/host/host.cpp

File: src/host/host.cpp [hunk 2]
```diff
@@ -45,17 +45,21 @@ class RPCHandler : virtual public RPCIf {
         assert(argc >= 2);
 
         char *oe_enclave_path = argv[1];
+        uint32_t simulate_flag = OE_ENCLAVE_FLAG_DEBUG;
```
Comment (adriandavila) [inline src/host/host.cpp:48]: We don't need debug mode for the experiment. Also it's no longer a simulate flag, moreso just a flag.

**Review by adriandavila — CHANGES_REQUESTED** [review]

File: src/host/host.cpp (hunk 1 above)
Comment (jasondu7297) [reply src/host/host.cpp]: Updated to use bitwise or

File: src/host/host.cpp (hunk 2 above)
Comment (jasondu7297) [reply src/host/host.cpp]: Updated flag name to `oe_flag`. Referring to the intel-sgx community: https://community.intel.com/t5/Intel-Software-Guard-Extensions/What-is-the-overhead-of-simulation-mode-VS-NO-SGX-at-all/m-p/1338169, enclaves should be launched in enclave-debug mode.

**Pull request merged**